packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -270,26 +270,51 @@ export class EditManager<
 	}
 
 	/**
-	 * Updates the `trunkBranches` map to reflect the fact that a new local commit was fast-forwarded to the trunk.
-	 * @param priorTrunkCommit - The trunk head prior to the fast-forward.
-	 * @param newId - The ID of the new trunk head.
+	 * Promote the oldest un-sequenced commit on the local branch to the head of the trunk.
+	 * @param sequenceId - The sequence id of the new trunk commit
+	 * @remarks This method is a performance optimization for the scenario where this client receives its own change back after sequencing.
```

Comment:
There should be a newline after the `@remarks`, right?

Reply:
I think it's fine to have remarks start on the same line (even multi-line remarks).